lib/rss.js

```diff
@@ -0,0 +1,192 @@
+const RSS = require('rss')
+const generateDiff = require('deep-diff').diff
+const fs = require('fs')
+
+function createLeadersHash(array) {
+  return array.reduce((hash, elem) => {
+    hash[elem.id] = elem
+    return hash
+  }, {})
+}
+
+function createOrgHash(array) {
+  let rank = 0
+  return array.reduce((hash, elem) => {
+    rank++
+    elem.rank = rank
+    elem.leaders = createLeadersHash(elem.leaders)
+    hash[elem.slug] = elem
+    return hash
+  }, {})
+}
+
+let current = {}
+try {
+  current = JSON.parse(
+    fs.readFileSync(`${__dirname}/../out/feed_items.json`).toString()
+  )
+} catch (e) {
+  current = {}
+}
+
+const buildInfo = JSON.parse(
+  fs.readFileSync(`${__dirname}/../out/scrape_info.json`).toString()
+)
+
+const oldData = JSON.parse(
+  fs.readFileSync(`${__dirname}/../out/data_old.json`).toString()
+)
+const newData = JSON.parse(
+  fs.readFileSync(`${__dirname}/../out/data.json`).toString()
+)
+
+const oldOrgs = createOrgHash(oldData)
+const newOrgs = createOrgHash(newData)
+
+console.log(oldOrgs)
+
+if (oldOrgs && newOrgs) {
+  const diffs = generateDiff(oldOrgs, newOrgs) || []
+  console.log(diffs)
```

Comment:
Do we need this? or is this for debugging, we might want a fancier log message or just throw this away completely.